src/pages/ProductList/ProductList.jsx

```diff
@@ -37,27 +22,12 @@ export const ProductList = () => {
                 <h1>NEW COLLECTIONS JUST ARRIVED!!!</h1>
             </div>
             <div className={`${styles.container}`}>
-                <FilterPane /> 
+                <FilterPane />
                 <div className={`${styles.bodyPane}`}>
                     <h1>{categories[2]?.categoryName}</h1>
                     <div className={`${styles.itemsList}`}>
-                        {filteredData(productState, products)?.map(({ _id, imgSrc, title, type, price, inStock, fastDelivery, onSale, discount }) => (
-                            <div key = {_id} className={`${styles.card1} card ecom`}>
-                                <div className="badge-h">
-                                    <i className="material-icons" onClick = {() => wishlistToggle({ _id, imgSrc, title, type, price, inStock, fastDelivery, onSale, discount })} >{wishlistButton}</i>
-                                    <img className="img-resp" src={imgSrc} alt="Products" />
-                                    <h5 className="head-badge card-badge warning">New</h5>
-                                </div>
-                                <div className="prod-details">
-                                    <p className="prod-cate">{type}</p>
-                                    <h4 className={`${styles.prodTitle} prod-head`}>{title}</h4>
-                                    {!inStock && <p className = {`${styles.itemNotAvailable} prod-cate`}>OUT OF STOCK</p>}
-                                    <p className="prod-det">Rs.{ discount } <span>Rs.{ price } </span></p>
-                                    {fastDelivery && <p className = {`${styles.itemAvailable} prod-cate`}>FAST DELIVERY</p>}
-                                    {onSale && <p className = {`${styles.itemAvailable} prod-cate`}>ON SALE</p>}
-                                    <div className="btn primary" onClick = { () => cartState.mycart.find(item => item._id === _id) ? (1 === 2) ? navigate("/cart") : navigate("/signin") : dispatch(addToCart({ _id, imgSrc, title, type, price, inStock, fastDelivery, onSale, discount })) }>Buy Now</div>
-                                </div>
-                            </div>
+                        {filteredData(productState, products)?.map(({_id, imgSrc, title, type, price, inStock, fastDelivery, onSale, discount, qty}) => (
+                        <ProductListComponent _id = {_id} imgSrc = {imgSrc} title = {title} type = {type} price = {price} inStock = {inStock} fastDelivery = {fastDelivery} onSale = {onSale} discount = {discount} qty = {qty} />
```

Comment:
Same multiline pattern is useful here as well for eg:
```
         <input
            type="text"
            placeholder="Search"
            className="tui__input--val"
            id="search"
          />

Reply:
sure Thanks